controller/users.go

```diff
@@ -186,6 +186,25 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 		var user *account.User
 		if identity.UserID.Valid {
 			user, err = appl.Users().Load(ctx.Context, identity.UserID.UUID)
+			/*
+
+				if tokenManager.IsServiceAccount(ctx){
```

Comment:
What if we just add a new action to design/account.go:
```
a.Action("updateuser", func() {
	a.Security("jwt")
	a.Routing(
		a.PATCH("/:id"),
	)
...
}
```
So we will have two actions:
The old one - PATCH /users (to be used by UI and will be removed after migration)
And a new one - PATCH /users/:identityID (to be used by auth service only)

And we should take care of creating new users in WIT too.

Reply:
Sounds fine to me. Thanks.
The old one should eventually be redirected ( instead of removed ) - but that's a different discussion.